core/templates/components/forms/validators/is-valid-url.filter.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Validator to check if input is nonempty.
+ */
+
+angular.module('oppia').filter('isValidUrl', [function() {
+  var IS_VALID_URL_REGEXP = /^((?!(mailto:)).)*$/;
```

Comment:
Hmm, can we have this regex as constant and use it in both frontend and backend?